README.md

```diff
@@ -144,7 +163,7 @@ Using Hubstaff TODOs only takes seconds to get right and there are a few reasons
 
 ## Why we use a time tracker
 
-It may seem weird, but actually RT is not the only consulting business out there. Traditionally lawyers bill their clients per hour and nobody blinks an eye :)
+It may seem weird, but actually RT is not the only consulting business out there. Traditionally lawyers bill their clients per hour and nobody blinks an eye :) Also a language teacher, therapist and nanny often bills like that.
 
 In fact many software houses nowadays bill their clients by the hour. In consequence, the income generated by a staff member is (almost) directly proportional to the amount of hours they spend working on the client projects (ok it's a big simplification because how sales activity is organized, but I don't want to digress here). Now unfortunately in the age of remote work (especially after the pandemic when everyone thinks they can work remotely) it's regrettably clear that some people can concentrate on their work much better than others. What I mean is that while some people can output 35 hours per week of focused work, other people can only output 10-12h. It would not be totally fair to compensate both of those groups equally, right?
 
```

Comment:
I think it would also be cool if we said that time tracker is used ONLY to 1) track time for further billing and 2) make snapshots of screen to recover after incorrect billing;
And those who are allergic to time tracker may:
* log their time manually and upload it to hubstaff weekly manually, or
* use hubstaff tracker but turn off screenshots

This will make non-tracker people rethink it.

Reply:
I don't think we are fully ok with either of those yet. We might be, but this requires a discussion and cannot be decided in this PR review.